lldb/test/API/lang/cpp/const_static_integral_member/TestConstStaticIntegralMember.py

```diff
@@ -74,9 +74,13 @@ def test(self):
         )
 
         # Test an enum with fixed underlying type.
-        self.expect_expr("A::scoped_char_enum_val", result_value="case2")
-        self.expect_expr("A::scoped_ll_enum_val_neg", result_value="case0")
-        self.expect_expr("A::scoped_ll_enum_val", result_value="case2")
+        self.expect_expr("A::scoped_char_enum_val", result_value="case2(2)")
+        self.expect_expr(
+            "A::scoped_ll_enum_val_neg", result_value="case0(-9223372036854775808)"
+        )
+        self.expect_expr(
+            "A::scoped_ll_enum_val", result_value="case2(9223372036854775807)"
+        )
```

Comment:
There are some XFAILed test-cases in this file. We should fix those up too so the XFAIL occurs for the intended reasons, and not an unexpected format